pre_commit_hooks/detect-secrets.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# Original: https://github.com/Yelp/detect-secrets
+
+set -eu
+
+readonly DEBUG=${DEBUG:-unset}
+if [ "${DEBUG}" != unset ]; then
+  set -x
+fi
+
+if ! command -v detect-secrets >/dev/null 2>&1; then
+  if command -v pip3 &>/dev/null; then
+    pip3 install detect-secrets==0.14.3
```

Comment:
If we use `language: python` then pre-commit can install this in a managed virtualenv and update it if needed - I think this will install it globally and it will not fail if the wrong version is there.

Reply:
Removed and using language python as suggested